benchmark/Project.toml

```diff
@@ -0,0 +1,8 @@
+[deps]
```

Comment:
It might also be worth adding a compat section to know if the changes are due to your package update or others, what do you think?

Reply:
Good idea!
But is PkgBenchmark disgned to be performed automatically for each new PR or not?

Reply:
Thanks for the pointer!

Reply:
If I understand correctly, `benchmark/benchmark.ml` is runned with the `benchmark/Project.toml` from the compared branches?